src/Standards/Generic/Tests/Functions/OpeningFunctionBraceKernighanRitchieUnitTest.3.inc

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// Intentional parse error (missing opening parenthesis).
+// This should be the only test in this file.
+// Testing that the sniff is *not* triggered.
+
+function missingOpeningParenthesis
```

Comment:
What is this testing ? The sniff is not looking at the open parenthesis, but at the opening curly.